includes/admin/class-wc-stripe-amazon-pay-controller.php

```
@@ -40,12 +40,15 @@ public function admin_scripts() {
 		wp_enqueue_script( 'wc-stripe-amazon-pay-settings' );
 
 		$stripe_settings = WC_Stripe_Helper::get_stripe_settings();
-		$params          = [
-			'key'                        => WC_Stripe_Mode::is_test() ? $stripe_settings['test_publishable_key'] : $stripe_settings['publishable_key'],
-			'locale'                     => WC_Stripe_Helper::convert_wc_locale_to_stripe_locale( get_locale() ),
-			'taxes_based_on_billing'     => wc_tax_enabled() && 'billing' === get_option( 'woocommerce_tax_based_on' ),
-			'is_button_style_overridden' => WC_Stripe_Helper::is_express_checkout_button_style_overridden(),
-		];
+		$params          = array_merge(
+			[
+				'key'                        => WC_Stripe_Mode::is_test() ? $stripe_settings['test_publishable_key'] : $stripe_settings['publishable_key'],
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Mode' in method 'admin_scripts'. (undefined)

(StaticAccess)